sdmetrics/reports/base_report.py

```diff
@@ -57,6 +59,28 @@ def validate(self, real_data, synthetic_data, metadata):
 
         self._validate_metadata_matches_data(real_data, synthetic_data, metadata)
 
+    @staticmethod
+    def convert_datetimes(real_data, synthetic_data, metadata):
+        """Try to convert all datetime columns to datetime dtype.
+
+        Args:
+            real_data (pandas.DataFrame):
+                The real data.
+            synthetic_data (pandas.DataFrame):
+                The synthetic data.
+            metadata (dict):
+                The metadata, which contains each column's data type as well as relationships.
+        """
+        for column, col_meta in metadata['columns'].items():
+            if col_meta['sdtype'] == 'datetime':
+                real_col = real_data[column]
+                synth_col = synthetic_data[column]
+                try:
+                    converted_cols = convert_datetime_columns(real_col, synth_col, col_meta)
+                    real_data[column], synthetic_data[column] = converted_cols
+                except Exception:
+                    continue
```

Comment:
I believe that recently we decided to stop using `staticmethods` on classes, I would encourage to move this to a utility function and call it when needed.

Reply:
Oh I see now, feel free to merge 👍🏻